packages/Card/src/components/Title/Title.styles.js

```diff
@@ -1,11 +1,14 @@
 import styled from "styled-components";
 import stylers from "@paprika/stylers";
 import tokens from "@paprika/tokens";
+import Heading from "@paprika/heading";
 
-export const titleStyles = styled.h1`
+export const Title = styled(Heading)`
   box-sizing: border-box;
   color: ${tokens.color.black};
+  font-weight: bold;
```

Comment:
Isn't it already bold by default? (Could be overridden if user passes an `isLight` prop).

Reply:
Oh! I see, because the `<h1>` is an exception and is always not bold.

I think actually you should try applying the `displayLevel={5}` prop as a default on the `<sc.Title>` and then see how much of this CSS you can actually get rid of.  

Maybe the `displayLevel={5}` should actually go _after_ the `...moreProps` so that it can't be overridden... 🤔